Ninano/Ninano/Controllers/EventTab/SearchResult/SearchResultViewController.swift

```diff
@@ -142,7 +149,7 @@ extension SearchResultViewController: DateDelivable, FilterButtonClickable {
         let koreanDate = date.convertDateToKoreanDate(.koreanDate)
         let button: UIButton = eventFilterButton.dateFilterButton
         button.setTitle(koreanDate, for: .normal)
-        button.configuration?.baseBackgroundColor = UIColor(hex: "D5DCF8")
+        button.configuration?.baseBackgroundColor = CustomColor.buttonLightRed
         button.configuration?.cornerStyle = .capsule
     }
```

Comment:
이거 전체를 하나의 func 으로 잡아서 넣어놔도 좋을 것 같아여!
- 여기서도 사실 굳이 따지면 더 나눌 수 있는데 그래도 시간이 없으니 하나로 퉁쳐서 묶어 아래에 따로 빼면 더 보기 쉽지 않을까 생각이 듭니다!
```suggestion
    func layout() {
        viewdidload() 안에 있는 코드들
    }
```

Reply:
감사합니다 !